plugins/alignments/src/BamAdapter/BamAdapter.ts

```diff
@@ -69,6 +69,16 @@ export default class BamAdapter extends BaseFeatureDataAdapter {
     }
   }
 
+  async getInfo(opts?: BaseOptions) {
+    const samHeader = await this.bam.getHeader(opts)
+    const rest = samHeader.filter((l: { tag: string }) => l.tag !== 'SQ')
+    // hide sqlines from info box by default
```

Comment:
Since this method may get used by things other than the info box, maybe a more generic comment would be useful

Reply:
I guess that is true. This could also be done by the info box itself so that this getInfo doesn't have to concern itself with this